Gems/Terrain/Code/Source/TerrainRenderer/Passes/TerrainClipmapComputePass.cpp

```diff
@@ -113,6 +113,11 @@ namespace Terrain
         return clipmapManager.HasMacroClipmapUpdate();
     }
 
+    bool TerrainMacroClipmapGenerationPass::ClipmapFeatureIsEnabled() const
```

Comment:
`ClipmapFeatureIsEnabled()` on the pass classes seems unnecessary. The terrain feature processor could just call `IsEnabled()` on the pass instead of `ClipmapFeatureIsEnabled()`.

Reply:
IsEnabled is overridden for another condition where there are no clipmap regions to update. So I have to make this one to tell if the feature is enabled.
Later we need a flag to enable clipmap at runtime and remove those.